### PR TITLE
suppress hadoop3 cve that seem not applicable to us

### PR DESCRIPTION
**Pull request by clintropolis**
follow-up to #14241

https://nvd.nist.gov/vuln/detail/CVE-2022-26612 - seems windows only?
https://nvd.nist.gov/vuln/detail/CVE-2023-25613 - seems like not a problem on druid side?

### CODE REVIEW

File: owasp-dependency-check-suppressions.xml [hunk 1]
```diff
@@ -806,4 +806,13 @@
     <!-- seems not applicable to plugin -->
     <cve>CVE-2022-45048</cve>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
+     file name: hadoop-client-runtime-3.3.5.jar
+     ]]></notes>
+    <!-- this one is windows only? https://nvd.nist.gov/vuln/detail/CVE-2022-26612 -->
```
Comment (gianm) [inline owasp-dependency-check-suppressions.xml:813]: This one says it was addressed in 3.2.3. Looks like we're using 3.3.5. I would think it's irrelevant just on that basis.

File: owasp-dependency-check-suppressions.xml [hunk 2]
```diff
@@ -806,4 +806,13 @@
...
+    <cve>CVE-2022-26612</cve>
+    <!-- this one seems to apply to backend server? https://nvd.nist.gov/vuln/detail/CVE-2023-25613 -->
```
Comment (gianm) [inline owasp-dependency-check-suppressions.xml:815]: This one is for Apache Kerby. Can we instead exclude it (if we're not using it) or upgrade it to 2.0.3 (if we are using it transitively)?

Comment (clintropolis) [reply owasp-dependency-check-suppressions.xml:815]: it's shaded inside the Hadoop client jar, so probably not?

Comment (gianm) [reply owasp-dependency-check-suppressions.xml:815]: Could we upgrade the hadoop client?

If we can't do any of this stuff, then I agree this appears to be about the backend server, which presumably we wouldn't be using in a client. So I think a suppression is OK. But it should be a last resort due to the severity. (Even if it's a false positive, suppressed high-sev false positives creates a bunch of work for downstream consumers that run their own scans.)

Ditch the "?" in the comment, though, once this analysis is complete. We need to be confident when suppressing things.

Comment (clintropolis) [reply owasp-dependency-check-suppressions.xml:815]: we are on the newest version unfortunately, 3.3.5 https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-client-runtime, so not an option either.

Comment (clintropolis) [reply owasp-dependency-check-suppressions.xml:815]: >Ditch the "?" in the comment, though, once this analysis is complete. We need to be confident when suppressing things.

heh, i'm not sure i'm ever fully confident when suppressing things, this stuff stresses me out

**Review by gianm — APPROVED** [review]

**Pull request merged**